src/Resolvers/Microsoft.DotNet.MSBuildSdkResolver/MSBuildSdkResolver.cs

```diff
@@ -170,7 +170,8 @@ public override SdkResult Resolve(SdkReference sdkReference, SdkResolverContext
             };
 
             //  First check if requested SDK resolves to a workload SDK pack
-            var workloadResult = workloadResolver.Resolve(sdkReference.Name, dotnetRoot, netcoreSdkVersion);
+            string userHome = null; // TODO
```

Comment:
I think what we'll want to do is to have a class that calculates the dotnet home folder, and it will be compiled into multiple projects as necessary.  We do that for a bunch of code in the MSBuildSdkResolver already in order to minimize the dependencies it has to take: https://github.com/dotnet/sdk/blob/4f46c0132783e0360d882dc5e4a082206534d21f/src/Resolvers/Microsoft.DotNet.MSBuildSdkResolver/Microsoft.DotNet.MSBuildSdkResolver.csproj#L63-L71